pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -456,11 +486,11 @@ contract TimelockAuthorizer is IAuthorizer, ReentrancyGuard {
         require(isRoot(msg.sender), "SENDER_IS_NOT_ROOT");
         bytes memory data = abi.encodeWithSelector(this.setPendingRoot.selector, newRoot);
 
-        // TODO: fix event action id
-
         // Since this can only be called by root, which is always a canceler for all scheduled executions, we don't
         // bother creating any new cancelers.
-        return _scheduleWithDelay(0x0, address(this), data, getRootTransferDelay(), executors);
+        uint256 scheduledExecutionId = _scheduleWithDelay(address(this), data, getRootTransferDelay(), executors);
+        emit RootChangeScheduled(scheduledExecutionId);
```

Comment:
```suggestion

        emit RootChangeScheduled(scheduledExecutionId);
```

Highly subjective, but I'd add whitespace here to separate the complex call from smaller bits like the event and return value :upside_down_face: